components/Layout.js

```diff
@@ -1,14 +1,14 @@
 import Navigation from './Navigation';
 import Footer from './Footer';
-import Cards from './Profiles';
+import Profiles from './Profiles';
 
 export default function Layout({ data, children }) {
   return (
     <div className="layout">
       <Navigation {...children} />
 
       <div className="page-content">
-        <Cards {...children} />
+        <Profiles {...children} />
```

Comment:
The profiles component must not be here, because the layout is the base layout of all pages and Profile is only in the index Page then put this line on `pages/index.js`
```suggestion
        {children}
```